### DISCUSSION TITLE
Small Reward Ideas

### DISCUSSION THREAD

**Discussion by mrezan12** [octoposprime/op-docs#21] [General]
What are your ideas about the small rewards that can be created as part of Small reward in Octopos Prime and where and when these rewards will be given?

**Comment by Sddilora** [comment]
@husamettinarabaci  proposed creating a sticker for this topic, and @Smnrgcl  and I both found the idea very appealing. Stickers could be designed for the first issue, pull request, and review. Furthermore, we could consider expanding options such as "first contributor" to make it more inclusive for juniors etc.

**Comment by Smnrgcl** [comment]
We can give stickers to individuals for '**first commit, first discussion, first comment'** parts.

**Reply by ebrarkesici** [reply]
We can also give topics that are not directly related to github: initial deploy, desk time limit broker
